sources/packages/backend/apps/api/src/route-controllers/education-program/models/education-program.dto.ts

```diff
@@ -151,6 +152,14 @@ export class EducationProgramAPIInDTO {
   programDeclaration: boolean;
   @Allow()
   fieldOfStudyCode: number;
+  /**
```

Comment:
Thanks for adding the comments. IMO properties comments should target each property.